app/invocation/invocation_not_found.tsx

```diff
@@ -15,19 +17,37 @@ export default class InvocationNotFoundComponent extends React.Component {
 
   render() {
     return (
-      <div className={this.props.authorized ? "state-page" : "login-interstitial"}>
-        {this.props.authorized && (
-          <div className="shelf">
+      <div className={this.props.isAuthenticated ? "state-page" : "login-interstitial"}>
```

Comment:
how does this stuff look if the user is logged out?

Reply:
It looks the same -- should hopefully be more clear after the refactor I applied

Reply:
so if a user is logged out, but they visit an invocation that doesn't exist - do they get a login screen or a not found screen?

Reply:
Ah, I see what you mean now. If the invocation doesn't exist, we want to show them the "not found" page. Made the changes so that it behaves this way now.